assignment_4.rb

```diff
@@ -0,0 +1,10 @@
+DAYS = %w[Mon Tue Wed Thu Fri Sat Sun]
```

Comment:
We normally put `freeze` after value of constant to avoid mutation. https://ruby-doc.org/core-2.7.3/Object.html#freeze-method